src/main/java/org/apache/ibatis/migration/Environment.java

```diff
@@ -113,7 +117,9 @@ public Environment(File file) {
       this.username = prop.getProperty(SETTING_KEY.username.name());
       this.password = prop.getProperty(SETTING_KEY.password.name());
 
-      this.hookBeforeUp = prop.getProperty(SETTING_KEY.hook_before_up.name());
+      this.hookBeforeNew = prop.getProperty(SETTING_KEY.hook_before_new.name());
```

Comment:
Follows current design. Importantly, there is no such thing as a `before_each` or `after_each` in the case of a new. So, I don't have analogs here for up/down